src/Compression/CompressionCodecDeflateQpl.h

```diff
@@ -24,22 +24,23 @@ class DeflateQplJobHWPool
     static DeflateQplJobHWPool & instance();
 
     qpl_job * acquireJob(UInt32 & job_id);
-    static void releaseJob(UInt32 job_id);
-    static const bool & isJobPoolReady() { return job_pool_ready; }
+    void releaseJob(UInt32 job_id);
+    const bool & isJobPoolReady() { return job_pool_ready; }
 
 private:
-    static bool tryLockJob(UInt32 index);
-    static void unLockJob(UInt32 index);
+    bool tryLockJob(UInt32 index);
+    void unLockJob(UInt32 index);
 
+    /// size of each job objects
+    UInt32 per_job_size;
     /// Maximum jobs running in parallel supported by IAA hardware
-    static constexpr auto MAX_HW_JOB_NUMBER = 1024;
+    UInt32 hw_jobs_max_number;
```

Comment:
Minor/cosmetic: `max` prefix for maximums is fairly common. What about just `max_hw_jobs`?

Reply:
Done